### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Inappropriate control type for the button in "Run on web app for containers dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1404]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish to web app for containers
4. "Run on web app for containers" dialog will open. 
5. In "Run on web app for containers" dialog, tab to "Azure Container Registry" ,"Web App for containers" , "resource Group" and "App Service Plan"  expand/collapse buttons and observe the control type using inspect

**Actual Result:**
Control type is pane
Is Expand/Collapse Property is "false"

**Narrator :**
Tab
Space
Space

**NVDA:**
Pane

**Expected Result:**
Control type should be button .
Is Expand/Collapse Property should be "true"

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app for containers-pane-control type-inspect](https://user-images.githubusercontent.com/22956130/35685064-aec5a3e0-078e-11e8-92a3-721b89ae3d78.jpg)
![run on web app for containers-pane-control type-narrator](https://user-images.githubusercontent.com/22956130/35685067-b1f05272-078e-11e8-8002-10386c1ca059.jpg)
![run on web app for containers-pane-control type-nvda](https://user-images.githubusercontent.com/22956130/35685074-b4eb9d1a-078e-11e8-90ed-5abcfdcdc7c6.jpg)

**Comment by Eskibear** [comment]
The Eclipse UI `org.eclipse.swt.widgets.ExpandBar` is out of the toolkit's control.

**Comment by Eskibear** [comment]
https://bugs.eclipse.org/bugs/show_bug.cgi?id=533980

**Comment by Eskibear** [comment]
linked to #1619

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. It's a legacy feature hardly ever used.
* **Quality of experience**: not impacted. All the widgets are expanded by default, users can complete the scenario without expand/collapse any of them.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by umasarath52** [comment]
Discussed with Ryan and approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749833
